erpnext/patches.txt

```diff
@@ -357,3 +357,4 @@ erpnext.patches.v15_0.create_advance_payment_status
 erpnext.patches.v14_0.migrate_gl_to_payment_ledger
 erpnext.stock.doctype.delivery_note.patches.drop_unused_return_against_index # 2023-12-20
 erpnext.patches.v14_0.set_maintain_stock_for_bom_item
+erpnext.patches.v14_0.update_posting_datetime_and_dropped_indexes
```

Comment:
We should do this at very top probably. 

Reason: some of the patches will runs code that reads the new field and it won't be present.

Reply:
Fixed